packages/app/src/runner/selector-playground/SelectorPlayground.vue

```diff
@@ -4,30 +4,30 @@
     class="border-t border-b bg-gray-50 border-gray-200 h-56px grid py-12px px-16px gap-12px grid-cols-[40px,1fr,auto] items-center "
   >
     <button
-      class="flex items-center justify-center h-full text-white transition duration-150 border rounded-md outline-none w-40px hover:default-ring"
```

Comment:
Since we're getting some UI changes along with the clipboard stuff, a before / after screenshot in the PR description will be really nice for people in the future trying to figure out where and why stuff changed.

Reply:
If there were any UI changes, they were unintentional. I will check.

Reply:
Automated reordering of classes is really a bummer...
Sry @BlueWinds there is really nothing changing here.

Reply:
Ah, gotcha.